### PR TITLE
FYLE-2v1r96m: Remove offline service methods -  Org User Settings

### PR DESCRIPTION
**Pull request by Dimple16**

### CODE REVIEW

File: src/app/core/services/offline.service.ts [hunk 1]
```diff
@@ -63,35 +63,9 @@ export class OfflineService {
...
-          );
-        } else {
-          return from(this.storageService.get('cachedOrgUserSettings'));
-        }
-      })
-    );
-  }
-
   @Cacheable()
   getOrgUserMileageSettings() {
```
Comment (Julias0) [inline src/app/core/services/offline.service.ts:67]: We should remove off this method from offline service and move it to a different service

File: src/app/core/services/offline.service.ts [hunk 2]
```diff
@@ -324,9 +298,9 @@ export class OfflineService {
 
   @Cacheable()
   getAllowedPaymentModes(): Observable<AccountType[]> {
```
Comment (Julias0) [inline src/app/core/services/offline.service.ts:300]: We should remove off this method from offline service and move it to a different service

**Review by Julias0 — CHANGES_REQUESTED** [review]
Added comments

File: src/app/core/services/offline.service.ts [hunk 3]
```diff
@@ -302,9 +271,9 @@ export class OfflineService {
...
-    return this.getOrgUserSettings().pipe(
-      map((orgUserSettings) => orgUserSettings?.payment_mode_settings?.allowed_payment_modes)
-    );
+    return this.orgUserSettingsService
```
Comment (Dimple16) [inline src/app/core/services/offline.service.ts:274]: removal of this method is taken care in another PR

**Review by Julias0 — APPROVED** [review]

**Pull request closed without merging**